runtime/Cargo.toml

```diff
@@ -167,6 +170,9 @@ std = [
     "pdex-migration/std",
 
     "frame-try-runtime/std",
+    "chainbridge/std",
```

Comment:
we need those pallets benchmarks to be included int runtime-benchmarks feature

Reply:
chainbridge is not benchmarked. Though I included AssetHandler.